package/MDAnalysis/analysis/dssp/pydssp_numpy.py

```diff
@@ -1,7 +1,7 @@
 """
-A re-implementation of DSSP algorithm [Kabsch1983]_, taken from `pydssp` v.0.9.0
-(https://github.com/ShintaroMinami/PyDSSP) by Shintaro Minami, distributed under
-MIT license.
+A re-implementation of DSSP algorithm :footcite:p:`Kabsch1983`, taken from
+`pydssp` v.0.9.0 (https://github.com/ShintaroMinami/PyDSSP) by Shintaro Minami,
```

Comment:
```suggestion
*pydssp* v.0.9.0 (https://github.com/ShintaroMinami/PyDSSP) by Shintaro Minami,
```

I am just nitpicking here...